source/manual/alerts/datagovuk-publish-healthcheck-not-ok.html.md

```diff
@@ -17,7 +17,3 @@ The healthcheck alert notifies us if one or both of these jobs are not running i
 the expected timeframe set. This means latest [dataset](https://ckan.publishing.service.gov.uk/dataset)
 or [publisher](https://ckan.publishing.service.gov.uk/publisher) updates are
 not appearing on data.gov.uk - see [Datasets published in CKAN are not appearing on Find](/manual/data-gov-uk-2nd-line.html#datasets-published-in-ckan-are-not-appearing-on-find)
-
-You can monitor the number of jobs in the Sidekiq queue
-via [the DGU dashboard](https://grafana-paas.cloudapps.digital/d/rk9fSapik/data-gov-uk-2nd-line?orgId=1)
```

Comment:
Worth checking with @kentsanggds if there are equivalent for this (and the dashboard URLs in `data/repos.yml`